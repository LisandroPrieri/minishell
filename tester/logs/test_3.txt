========== Test 3 ==========
Command: 
======================================
Output: bash: line 1: infile1: No such file or directory
======================================
Return Value: 
========== End of Test 3 ==========
No expected outfile found: ./expected_outputs/test_3.txt
