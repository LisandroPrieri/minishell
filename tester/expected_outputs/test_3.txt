========== Test 3 ==========
Command: cat <infile1 <infile2 <infile3 infile1 infile2 infile3
======================================
Output: abcdefghi
======================================
Return Value: 0
========== End of Test 3 ==========
